notebooks/MAST/plot_exoplanet_lightcurves/plot_exoplanet_lightcurves.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
```

Comment:
<p>Spitzer is a mess. I don't think it has any files listed as "timeseries"</p><p> </p><p>Limb-darkening/atmospheric absorption would be interesting but somewhat complicated here. Perhaps as an exercise for the reader, to be included at the bottom of the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>